modules/parrableIdSystem.js

```diff
@@ -242,6 +259,10 @@ function fetchId(configParams, gdprConsentData) {
               if (responseObj.ibaOptout === true) {
                 newParrableId.ibaOptout = true;
               }
+              if (responseObj.tpcSupport !== undefined) {
+                newParrableId.tpcSupport = responseObj.tpcSupport;
+                newParrableId.tpcUntil = Date.now() + responseObj.tpcSupportTtl;
```

Comment:
Similarly to handling tpcSupport and checking tpcUntil in readCookie() I think we should do the ttl to date computation in writeCookie(); 

Also, we should store the value in seconds, since milliseconds is unnecessarily large.

Reply:
I don't want to be passing the `tpcSupportTtl` value through all the methods. I'd rather let it as it is.